internal/pipeline/executor.go

```diff
@@ -139,15 +142,32 @@ func (p *pkg) resolveSources() []string {
 			// include only this pkg sources
 			pkgPaths = append(pkgPaths, p.Path())
 		case sourceAllSubPkgs:
-			// TODO(droot): figure out how to enumerate sub packages for current pkgPath.
-			// For now including current pkg
+			// For now including current pkg and subpkgs in current directory
 			pkgPaths = append(pkgPaths, p.Path())
+			files, err := ioutil.ReadDir(p.Path())
+			if err != nil {
+				return nil, err
+			}
+			for _, f := range files {
+				if f.IsDir() {
+					// A directory is a package if it has a Kptfile.
+					// This may change as the concept of a package is expanded such that
+					// every directory is its own kpt package
+					absolutePath := path.Join(p.Path(), f.Name())
+					subPaths, err := pathutil.DirsWithFile(absolutePath, "Kptfile", false)
```

Comment:
We should use the constant for the name of the Kptfile. I think it is `kptfile.KptFileName`

Reply:
done, thanks